styles.css

```diff
@@ -1 +1,84 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+/*styling*/
+body {background-color: #F0E9E4}
+
+/*styling*/
+section {
+    border-style: solid;
+    border-width: 2px;
+    border-radius: 5px;
+    border-color: #304D6D;
+    background-color: #82A0BC;
+    padding: 2px;
+}
+
+/*grid start*/
+div {
+    grid-gap: 16px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
```

Comment:
Je kan hier ook gebruik maken van `grid-template-columns: repeat(3, 1fr);`
(Beetje hypocriet, heb het in mijn eigen project ook niet gedaan :D)